cli/lib/kontena/cli/stacks/yaml/opto/prompt_resolver.rb

```diff
@@ -50,7 +50,7 @@ def enum
       end
 
       def bool
-        prompt.yes?(question_text, default: option.default == false ? false : true)
+        prompt.yes?(question_text, default: option.default.nil? || option.default)
```

Comment:
It seems like `default: false` in the YAML turns into `option.default = nil` :speak_no_evil: